src/lib/configDataSeeder.js

```diff
@@ -0,0 +1,134 @@
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * Claudio Viola <claudio.viola@modusbox.com>
+ --------------
+******/
+'use strict'
+
+// const LedgerAccountTypeModel = require('../models/ledgerAccountType/ledgerAccountType')
+const ErrorHandler = require('@mojaloop/central-services-error-handling')
+const Config = require('./config')
+const SettlementService = require('../domain/settlement')
+const SettlementModel = require('../models/settlement/settlementModel')
+
+// const LedgerAccountTypesService = require('../domain/ledgerAccountTypes')
+const Db = require('./db')
+
+const SETTLEMENT_MODELS_ALIASES = {
+  CGS: {
+    name: 'CGS',
+    settlementGranularity: 'GROSS',
+    settlementInterchange: 'BILATERAL',
+    settlementDelay: 'IMMEDIATE',
+    requireLiquidityCheck: true,
+    ledgerAccountType: 'POSITION',
+    autoPositionReset: false,
+    settlementAccountType: 'SETTLEMENT'
+  },
+  DEFERREDNET: {
+    name: 'DEFERREDNET',
+    settlementGranularity: 'NET',
+    settlementInterchange: 'MULTILATERAL',
+    settlementDelay: 'DEFERRED',
+    requireLiquidityCheck: true,
+    ledgerAccountType: 'POSITION',
+    autoPositionReset: true,
+    settlementAccountType: 'SETTLEMENT'
+  },
+  INTERCHANGEFEE: {
+    name: 'INTERCHANGEFEE',
+    settlementGranularity: 'NET',
+    settlementInterchange: 'MULTILATERAL',
+    settlementDelay: 'DEFERRED',
+    requireLiquidityCheck: false,
+    ledgerAccountType: 'INTERCHANGE_FEE',
+    autoPositionReset: true,
+    settlementAccountType: 'INTERCHANGE_FEE_SETTLEMENT'
+  }
+}
+
+/**
+ * [initializeSeedData Adds configurable seeds data]
+ */
+async function initializeSeedData () {
+  const knex = Db.getKnex()
+  return knex.transaction(async trx => {
+    try {
+      // await initializeLedgerAccountTypes(trx)
+      await initializeSettlementModels(trx)
+
+      await trx.commit
+    } catch (err) {
+      await trx.rollback
+      throw ErrorHandler.Factory.reformatFSPIOPError(err)
+    }
+  })
+}
+
+async function initializeSettlementModels (trx) {
+  const settlementModelNamesToCreate = Config.SETTLEMENT_MODELS.map(item => {
+    return SETTLEMENT_MODELS_ALIASES[item].name
+  })
+  let existingSettlementModelsNames = await SettlementModel.getSettlementModelsByName(settlementModelNamesToCreate, trx)
+  existingSettlementModelsNames = existingSettlementModelsNames.map(record => record.name)
+  const missingAccountTypes = Config.SETTLEMENT_MODELS.filter(item => {
```

Comment:
What if `Config.SETTLEMENT_MODELS` isn't set? Config should be backwards compatible. Maybe we can have some sensible defaults, or _explicitly_ make a breaking change.